src/Files.App/Views/ContentLayouts/DetailsLayoutBrowser.xaml

```diff
@@ -149,7 +150,7 @@
 				</Setter>
 			</Style>
 
-			<Style TargetType="uc:DataGridHeader">
+			<Style BasedOn="{StaticResource DefaultDataGridHeaderStyle}" TargetType="uc:DataGridHeader">
```

Comment:
It might be better to do this in a separate PR so we can review the change on it's own.